### PR TITLE
Bump to citus 11.1.6

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

**Review by onurctirtir — DISMISSED** [review]

File: citus.spec [hunk 1]
```diff
@@ -8,11 +8,13 @@ Summary:	PostgreSQL-based distributed RDBMS
 Name:		%{sname}%{?pkginfix}_%{pgmajorversion}
 Provides:	%{sname}_%{pgmajorversion}
 Conflicts:	%{sname}_%{pgmajorversion}
-Version:	11.2.1.citus
+
+Version:	11.1.6.citus
```
Comment (hanefi) [inline citus.spec]:
```suggestion
Version:	11.1.6.citus
```

File: citus.spec [hunk 2]
```diff
@@ -8,11 +8,13 @@ Summary:	PostgreSQL-based distributed RDBMS
...
 Release:	1%{dist}
 License:	AGPLv3
 Group:		Applications/Databases
-Source0:	https://github.com/citusdata/citus/archive/v11.2.1.tar.gz
+Source0:	https://github.com/citusdata/citus/archive/v11.1.6.tar.gz
+
```
Comment (hanefi) [inline citus.spec]:
```suggestion
Source0:	https://github.com/citusdata/citus/archive/v11.1.6.tar.gz
```

**Review by hanefi — DISMISSED** [review]
please remove extra lines before merging

File: citus.spec (hunk 2 above)
Comment (hanefi) [reply citus.spec]: I guess there is something wrong with the Github interface. Can you remove the new empty line from here @gurkanindibay ?

Comment (gurkanindibay) [reply citus.spec]: Ok I can not see it from the interface. Fixed it from my local and pushed

**Review by hanefi — APPROVED** [review]

**Pull request merged**